root/static/scripts/release-editor/fields.js

```diff
@@ -298,6 +298,7 @@ class Medium {
         this.name = ko.observable(data.name);
         this.position = ko.observable(data.position || 1);
         this.formatID = ko.observable(data.formatID);
+        this.formatUnknownToUser = ko.observable(Boolean(data.id));
```

Comment:
But only if there's no format ID selected. So it should be `Boolean(data.id && ! data.formatID)`, right?

Reply:
I'm not sure about Knockout's architecture here - is the `Medium` object constructed before or after the UI is made? It seems to be that it would be before, so this wouldn't be necessary, but I'm not sure.